src/reducers/query.ts

```diff
@@ -146,6 +154,49 @@ export default {
           errMsg: { $set: action.payload },
           polling: { $set: false },
         }),
+      [ActionTypes.COLUMN_STATS_FAILURE]: (state, action: any) =>
+        immutable(state, {
+          error: { $set: true },
+          errMsg: { $set: action.payload },
+          polling: { $set: false },
+        }),
+      [ActionTypes.COLUMN_STATS_TEXT_SUCCESS]: (state, action: any) => {
+        const values = action.payload.queryResults.data.values;
```

Comment:
Might make sense to just do this now so it doesn't keep squawking at us
```suggestion
        const { values } = action.payload.queryResults.data;
```

Reply:
Ah, thanks!